src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -4769,6 +4769,9 @@ int32_t NaClSysExecv(struct NaClAppThread *natp, char const *path, char *const *
   for (int i = 0; i < new_argc; i++) {
     child_argv[i + 3] = new_argv[i] ? strdup(new_argv[i]) : NULL;
   }
```

Comment:
I think a newline between the for loop and if statement is clearer stylistically to separate them.